v3/plugins/cdr/code-server/2.1523-vsc1.38.1/meta.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v2
+publisher: cdr
+name: code-server
+version: 2.1523-vsc1.38.1
+type: Che Editor
+displayName: code-server
+title: Visual Studio Code - Web (code-server)
+description: A open source distribution of Visual Studio Code as a cloud IDE
+icon: https://raw.githubusercontent.com/sr229/discord-open-source/master/logos/code-server.svg 
+category: Editor
+repository: https://github.com/cdr/code-server
+firstPublicationDate: "2019-09-08"
+spec:
+  endpoints:
+   -  name: "code-server"
+      public: true
+      targetPort: 9000
+      attributes:
+        protocol: http
+        type: ide
+  containers:
+   - name: code-server
+     image: "docker.io/chinodesuuu/coder:2.1523-vsc1.38.1-che"
```

Comment:
@sr229 Thanks for the PR! We try to avoid linking personal dockerhub repositories. Would you be able to push the image to `docker.io/codercom` repo instead?

Reply:
For now that isn't possible, but I'll ask if I can push to the Docker namespace soon.

Reply:
ack